src/main/java/Duke.java

```diff
@@ -1,10 +1,239 @@
+            error(1);
+        }
+    }
+
+    /**
+     * Check respective task as done
+     * @param index - index of list
+     */
+    public static void checkTask(int index){
+        if (list[index].isDone) {
+            error(4);
+        } else {
+            System.out.println("Good Job, I will mark this as done!");
+            list[index].markAsDone();
+            System.out.println("[" + list[index].getType() + "] ["
+                + list[index].getStatusIcon() + "] " + list[index].getName()
+                + list[index].getDate());
+            PrintOutput.printBorder();
+        }
+    }
+
+    /**
+     * Print the number of task in list
+     */
+    public static void printNoOfTask() {
+        System.out.print("You have " + LIST_COUNTER + " task");
+        if(LIST_COUNTER > 1){
+            System.out.print("s");
+        }
+        System.out.print(" in total!\n");
+        PrintOutput.printBorder();
+    }
+
+    /**
+     * print List
+     */
+    public static void printList(){
+        if (LIST_COUNTER > 0) {
+            System.out.println(" LIST");
+            for (int i = 0; i < LIST_COUNTER; i++) {
+                System.out.println(i + 1 +  ". [" + list[i].getType() +  "] " + "["
+                    +list[i].getStatusIcon() + "] " + list[i].getName()
+                    + list[i].getDate());
+            }
+            PrintOutput.printBorder();
+        } else {
+            error(3);
+        }
+    }
+
+
+    /**
+     * Returns error code message
+     * Error Code 1: done command in wrong format
+     * Error Code 2: done Index is more than size of array
+     * Error Code 3: Timetable is empty
+     * Error Code 4: Task already marked done
+     * Error Code 5: Invalid Command format
+     * @param code - error code
+     */
+    public static void error(int code) {
+        switch (code) {
+            case 1:
+                System.out.println("Error! You must enter an integer after"
+                    + " \"done\"!");
+                PrintOutput.printBorder();
+                break;
+            case 2:
+                System.out.println("Error! You do not have that "
+                    + "many items in your list!");
+                PrintOutput.printBorder();
+                break;
+            case 3:
+                System.out.println("Your list is empty! Add something!");
+                PrintOutput.printBorder();
+                break;
+            case 4:
+                System.out.println("You have already marked it as Done!");
+                PrintOutput.printBorder();
+                break;
+            case 5:
+                System.out.println("¯\\_(ツ)_/¯ That is an invalid format!");
+                System.out.println("Enter HELP for commands!");
+                PrintOutput.printBorder();
+                break;
+            case 6:
+                System.out.println("List is full!");
+                PrintOutput.printBorder();
+                break;
```

Comment:
The `case` should be on a same indentation level as `switch`.  Refer to the coding standard's `The switch statement should have the following form` section.